codespell_lib/data/dictionary.txt

```diff
@@ -4475,7 +4475,6 @@ cloacks->cloaks
 cloberring->clobbering
 clocksourc->clocksource
 clockwíse->clockwise
-clock_getttime->clock_gettime
```

Comment:
You can add getttime instead on it's own can't you?

Reply:
Did the tests flag this up or did you check?

Reply:
It was already there (and I think was automatically deduped by the check) 😄

Reply:
:facepalm: I swear I checked at least one of these!